vispy/visuals/rectangle.py

```diff
@@ -181,7 +181,7 @@ def radius(self, radius):
                                  " (got %s of length %d)" % (type(radius),
                                                              len(radius)))
 
-            if (radius > np.full(4, hw)).all():
+            if (radius > np.ones(4) * hw).all():
```

Comment:
Yeah, that's true actually. I wasn't thinking too hard about what was going on. Feel free to open a PR if it bothers you to that level (it's not quite there for me), otherwise we can try to get it on a subsequent one.